gems/sorbet-runtime/test/types/configuration.rb

```diff
@@ -328,31 +328,5 @@ def @mod.foo(a)
         end
       end
     end
-
-    describe 'vm_prop_serde' do
-      it 'raises errors if no runtime support exists' do
-        # This test is checking that we get errors from serialization/deserialization
-        # when the VM we're running on doesn't support VM-backed serialization/deserialization
-        # routines, so bail if the VM does support such routines.
-        return if T::Props::Private::DeserializerGenerator.singleton_class.method_defined?(:generate2)
-
-        was_enabled = T::Configuration.use_vm_prop_serde?
-
-        begin
-          T::Configuration.enable_vm_prop_serde
-          c = Class.new(T::Struct) do
-            prop :p, Integer
-          end
-
-          i = c.new(p: 5)
-          e = assert_raises(NoMethodError) do
-            s = i.serialize
-          end
-          assert_equal(:generate2, e.name)
-        ensure
-          T::Configuration.disable_vm_prop_serde unless was_enabled
-        end
-      end
-    end
```

Comment:
Shouldn't this test still pass as written? Why delete it? At the least, why have we not replaced it with another test?

Reply:
Because `T::Configuration.enable_vm_prop_serde` now hard-asserts if the VM doesn't support the necessary bits, so the test falls over.  I don't really think it's worth dancing around that to placate the test.

Reply:
Ah sorry I missed that line. But we can still add a new test, right?

Reply:
We could add:

```ruby
describe 'enable_vm_prop_serde' do
  it "fails if the VM doesn't support it" do
    return if T::Configuration.can_enable_vm_prop_serde?

    assert_raises(RuntimeError) do
      T::Configuration.enable_vm_prop_serde
    end
  end
end
```

Or somesuch?

Reply:
Sure. Even better would be to run `enable_vm_prop_serde` in both cases though, right? In one ensure that it raises, in the other ensure that it doesn't